### ISSUE TITLE
Bug: Inline code in link title in footnotes is broken

### ISSUE THREAD

**Issue by msiemens** [TryGhost/Ghost#5237]
The following markdown 

``` md
[^1]: [`Something`](http://example.com)
```

renders as this HTML:

``` html
<li class="footnote" id="fn:1"><p>[</p>

<p>undefined](http://example.com) <a href="#fnref:1" title="return to article">↩</a></p></li>
```

Example:

![](http://i.imgur.com/cQCTvrI.png)

I'm using Ghost v0.6.2

**Comment by msiemens** [comment]
The same happens with this markdown:

```md
[^1]: [<code>Something</code>](http://example.com)
```

**Comment by ErisDS** [comment]
This bug should be resolved by #5167
